clang/lib/Headers/emmintrin.h

```diff
@@ -2099,9 +2099,11 @@ static __inline__ __m128i __DEFAULT_FN_ATTRS _mm_add_epi64(__m128i __a,
 }
 
 /// Adds, with saturation, the corresponding elements of two 128-bit
-///    signed [16 x i8] vectors, saving each sum in the corresponding element of
-///    a 128-bit result vector of [16 x i8]. Positive sums greater than 0x7F are
-///    saturated to 0x7F. Negative sums less than 0x80 are saturated to 0x80.
+///    signed [16 x i8] vectors, saving each sum in the corresponding element
+///    of a 128-bit result vector of [16 x i8].
+///
```

Comment:
Can we remove the blank line to make to more compact?

Reply:
The [description of brief](https://www.doxygen.nl/manual/commands.html#cmdbrief) says "A brief description ends when a blank line or another sectioning command is encountered." About the only command that would make sense here is \par which has to be on a line by itself, so the result would not be more compact.

Reply:
That's exactly the reason for my suggestion. By making it more compact, I actually mean we should make them one brief description instead of two. The second paragraph is not another \brief or \details. It's jsut supplement to the first paragraph.

Reply:
Adding the blank line makes the brief section more compact, so I'm not clear what you want.

In our documentation, the first paragraph appears at the top of the documentation page for the function; both paragraphs will appear in the full description after the synopsis. No information is lost. I believe the detail about how saturation works is not needed in the brief description.

Are you saying the detail about saturation belongs in the brief description? Or something else?

Reply:
> Are you saying the detail about saturation belongs in the brief description?

Yes. I though otherwise the full description may turn into the second paragraph only. I'm not a doxygen user, so the explanation sounds good to me.